src/fileTransformers/deployToVercelButton.ts

```diff
@@ -0,0 +1,21 @@
+import { FileTransformer } from '../fileTransformer/fileTransformer'
+
+export const deployToVercelButton: FileTransformer = (params) => {
+  const { file, parameters } = params
+
+  let content = file.content
+
+  if (
+    parameters.repositoryHandle !== 'undefined' &&
+    parameters.repositoryOwner !== 'undefined' &&
```

Comment:
Why quoted undefined?

Reply:
I commented about this above. I get a run time error if I don't set these defaults as strings, even with the `?` in `generator/cli/generate-type-script.ts` Line 257.
Then I added this conditional to only display the deploy to vercel button when we explicitly pass `repositoryOwner` and `repositoryHandle` in. Is that correct?